WhiteDentalClinic/WhiteDentalClinic.Api/Controllers/MedicalServicesController.cs

```diff
@@ -6,32 +6,32 @@
 
 namespace WhiteDentalClinic.Api.Controllers
 {
-    public class MedicalServiceController : ApiController
+    public class MedicalServicesController : ApiController
     {
         private readonly IMedicalServiceService _medicalService;
-        public MedicalServiceController(IMedicalServiceService medicalService) 
+        public MedicalServicesController(IMedicalServiceService medicalService) 
         {
             _medicalService= medicalService;
         }
 
-        [HttpGet("all")]
+        [HttpGet]
         public ActionResult<IEnumerable<ResponseMedicalServices>> GetAllMedicalServices()
         {
             try
             {
-                return Ok(ApiGenericsResult<IEnumerable<ResponseMedicalServices>>.Success(_medicalService.GetAllMedicalServices()));
+                return Ok(ApiGenericsResult<IEnumerable<ResponseMedicalServices>>.Success(_medicalService.GetAll()));
             }
             catch(Exception ex)
             {
                 return BadRequest(ApiGenericsResult<ResponseMedicalServices>.Failure(new[] { $"{ex.Message}" }));
             }
         }
-        [HttpGet("all/dentist{requestDentistId}")]
-        public ActionResult<IEnumerable<ResponseMedicalServices>> GetAllMedicalServicesByDentistId(Guid requestDentistId)
+        [HttpGet("dentist{DentistId}")]
```

Comment:
Add slash before "{DentistId}". Also, keep consistency in mind - all the other ids in the other controllers are named starting lowercase only this one is uppercase. This is clearly a sign it should be "dentists/{dentistId}" or uppercase first letter should be used everywhere else.